cibyl/models/ci/environment.py

```diff
@@ -47,13 +47,12 @@ def __init__(self, name, systems=None):
 
     def add_system(self,
                    name: str, system_type: str,
-                   sources: list = None, enabled: bool = True, **kwargs):
+                   sources: list = None, enabled: bool = True):
```

Comment:
I think it would be simpler to leave here the kwargs and remove them when creating the systems in the system factory

Reply:
Yeah, you are right about it. Made some modifications.